README.md

```diff
@@ -313,13 +313,15 @@ Service name    | Units                                     | Description
 
 #### Interfaces
 
-Type | Interface                     | Value type | Properties                         | Description
------|-------------------------------|------------|------------------------------------|-------------
-in   | cmd.meter.get_report          | string     | direction                          | Value - is a unit. May not be supported by all meters.
-in   | cmd.meter.reset               | null       |                                    | Resets all historical readings.
-out  | evt.meter.report              | float      | unit, prv_data, delta_t, direction |
-out  | evt.meter_ext.report          | float_map  |                                    | [Extended meter report](#extended-report-object) with up to 17 data points
-in   | cmd.meter_ext.get_report      | null       |                                    | Request extended report
+Type | Interface                     | Value type | Properties              | Description
+-----|-------------------------------|------------|-------------------------|-------------
+in   | cmd.meter.get_report          | string     |                         | Value - is a unit. May not be supported by all meters.
+in   | cmd.meter.get_export_report   | string     |                         | Value - is an export unit. May not be supported by all meters.
+in   | cmd.meter.reset               | null       |                         | Resets all historical readings.
+out  | evt.meter.report              | float      | unit, prv_data, delta_t | Returns consumption meter report.
+out  | evt.meter.export_report       | float      | unit, prv_data, delta_t | Returns production meter report.
```

Comment:
Maybe unify usage of `import/export` and `consumption/production`, use either one pair throughout entire service or use `import/consumption` and `export/production` together to give the best explanatory value.

Reply:
using pair together